core/src/main/java/org/mskcc/cbio/maf/MafUtil.java

```diff
@@ -208,79 +192,57 @@
     private int normalVafIndex = -1;
 
     // default Oncotator column indices
```

Comment:
* Removed several oncotator fields (+ a couple others) that are not referenced/used when constructing `mutation`, `mutation_event` objects for import. 
* Removed corresponding globals referring to their index val in the MAF header.
* Removed corresponding references to `MafRecord` setters